### PR TITLE
updates Readme.md and replaces parallelshell with npm-run-all

### PR DESCRIPTION
**Pull request by frometor**
fixes https://github.com/zalando/zappr/issues/545

### CODE REVIEW

**Comment by fokusferit** [comment]
👍

File: MAINTAINERS [hunk 1]
```diff
@@ -1,2 +1,4 @@
 Team MACHINERY <team-machinery@zalando.de>
 ferit.topcu@zalando.de
+damian.garcia@zalando.de
+erik.zocher@zalando.de
```
Comment (fokusferit) [inline MAINTAINERS]: 🎆

**Comment by lotharschulz** [comment]
👍

File: README.md [hunk 2]
```diff
@@ -16,56 +16,101 @@ halt "rogue" pull requests before they're merged into the master branch.
...
-lt -s myzappr -p 3000
-~~~
+Go to `https://<your-app-name>.localtunnel.me` and do things :)
 
-4. Set your OAuth application URLs to `https://myzappr.localtunnel.me` in the GitHub settings
+##### Using docker compose:    
+  ~~~ shell
+  docker-compose up
+  ~~~
+  @TOOO: how to access the local zappr installation
```
Comment (fokusferit) [inline README.md]: @lotharschulz what do you mean with how to access local zappr installation? docker-compose up should run the databases and do npm start -> therefore on localhost you should be able to access it, shouldn't it ?

File: README.md [hunk 3]
```diff
@@ -110,6 +159,91 @@ Then you can do:
 * `NPM_BUILD_INSIDE_CONTAINER` "true" to build inside a container
 * `DOCKER_ARTIFACT_FILE` file to store docker artifact name in
 
+#### Trouble shooting:
```
Comment (fokusferit) [inline README.md]: +1 for troubleshooting section :)

File: README.md (hunk 2 above)
Comment (lotharschulz) [reply README.md]: I hope so! `docker-compose up` did no work for me and I could neither find out why nor fix it, so I left the @TODO. Does it work for you?

**Comment by lotharschulz** [comment]
`docker-compose up` still fails for me unfortunately with:

```
zappr-app        | npm info it worked if it ends with ok
zappr-app        | npm info using npm@4.1.2
zappr-app        | npm info using node@v6.9.4
zappr-app        | npm info lifecycle zappr@0.1.0-alpha.1~prestart: zappr@0.1.0-alpha.1
zappr-app        | npm info lifecycle zappr@0.1.0-alpha.1~start: zappr@0.1.0-alpha.1
zappr-app        | 
zappr-app        | > zappr@0.1.0-alpha.1 start /opt/zappr
zappr-app        | > node dist/server/server.min.js
zappr-app        | 
zappr-app        | 
zappr-app        | npm info lifecycle zappr@0.1.0-alpha.1~start: Failed to exec start script
zappr-app        | npm ERR! Linux 4.9.125-linuxkit
zappr-app        | npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
zappr-app        | npm ERR! node v6.9.4
zappr-app        | npm ERR! npm  v4.1.2
zappr-app        | npm ERR! code ELIFECYCLE
zappr-app        | npm ERR! zappr@0.1.0-alpha.1 start: `node dist/server/server.min.js`
zappr-app        | npm ERR! Exit status 1
zappr-app        | npm ERR! 
zappr-app        | npm ERR! Failed at the zappr@0.1.0-alpha.1 start script 'node dist/server/server.min.js'.
zappr-app        | npm ERR! Make sure you have the latest version of node.js and npm installed.
zappr-app        | npm ERR! If you do, this is most likely a problem with the zappr package,
zappr-app        | npm ERR! not with npm itself.
zappr-app        | npm ERR! Tell the author that this fails on your system:
zappr-app        | npm ERR!     node dist/server/server.min.js
zappr-app        | npm ERR! You can get information on how to open an issue for this project with:
zappr-app        | npm ERR!     npm bugs zappr
zappr-app        | npm ERR! Or if that isn't available, you can get their info via:
zappr-app        | npm ERR!     npm owner ls zappr
zappr-app        | npm ERR! There is likely additional logging output above.
zappr-app        | 
zappr-app        | npm ERR! Please include the following file with any support request:
zappr-app        | npm ERR!     /opt/zappr/npm-debug.log
zappr-app exited with code 1
```

**Comment by fokusferit** [comment]
@lotharschulz could you try out 
`docker-compose up zappr` ?

And I think you need to provide github credentials in de docker compose file. Empty stuff needs to be defined + DM_IP should be defined in the terminal environment

**Comment by lotharschulz** [comment]
```
$ docker-compose up zappr
WARNING: The DM_IP variable is not set. Defaulting to a blank string.
zappr-postgres-dev is up-to-date
Starting zappr-app ... done
Attaching to zappr-app
zappr-app        | npm info it worked if it ends with ok
zappr-app        | npm info using npm@4.1.2
zappr-app        | npm info using node@v6.9.4
zappr-app        | npm info lifecycle zappr@0.1.0-alpha.1~prestart: zappr@0.1.0-alpha.1
zappr-app        | npm info lifecycle zappr@0.1.0-alpha.1~start: zappr@0.1.0-alpha.1
zappr-app        | 
zappr-app        | > zappr@0.1.0-alpha.1 start /opt/zappr
zappr-app        | > node dist/server/server.min.js
zappr-app        | 
zappr-app        | 
zappr-app        | npm info lifecycle zappr@0.1.0-alpha.1~start: Failed to exec start script
zappr-app        | npm ERR! Linux 4.9.125-linuxkit
zappr-app        | npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
zappr-app        | npm ERR! node v6.9.4
zappr-app        | npm ERR! npm  v4.1.2
zappr-app        | npm ERR! code ELIFECYCLE
zappr-app        | npm ERR! zappr@0.1.0-alpha.1 start: `node dist/server/server.min.js`
zappr-app        | npm ERR! Exit status 1
zappr-app        | npm ERR! 
zappr-app        | npm ERR! Failed at the zappr@0.1.0-alpha.1 start script 'node dist/server/server.min.js'.
zappr-app        | npm ERR! Make sure you have the latest version of node.js and npm installed.
zappr-app        | npm ERR! If you do, this is most likely a problem with the zappr package,
zappr-app        | npm ERR! not with npm itself.
zappr-app        | npm ERR! Tell the author that this fails on your system:
zappr-app        | npm ERR!     node dist/server/server.min.js
zappr-app        | npm ERR! You can get information on how to open an issue for this project with:
zappr-app        | npm ERR!     npm bugs zappr
zappr-app        | npm ERR! Or if that isn't available, you can get their info via:
zappr-app        | npm ERR!     npm owner ls zappr
zappr-app        | npm ERR! There is likely additional logging output above.
zappr-app        | 
zappr-app        | npm ERR! Please include the following file with any support request:
zappr-app        | npm ERR!     /opt/zappr/npm-debug.log
zappr-app exited with code 1
```

which is a similar error to:

```
$ npm start                                                        

> zappr@0.1.0-alpha.1 start /Users/lschulz/zalando/dev/zappr
> NODE_ENV=production node dist/server/server.min.js

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zappr@0.1.0-alpha.1 start: `NODE_ENV=production node dist/server/server.min.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zappr@0.1.0-alpha.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/lschulz/.npm/_logs/2019-01-09T08_53_23_802Z-debug.log
```
Please note: I adapted package.json and set
`"start": "NODE_ENV=production node dist/server/server.min.js"`

```
$ cat /Users/lschulz/.npm/_logs/2019-01-09T08_53_23_802Z-debug.log
....
8 verbose lifecycle zappr@0.1.0-alpha.1~start: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/lschulz/zalando/dev/zappr/node_modules/.bin:/Users/lschulz/zalando/dev/go/bin:/usr/local/opt/gnu-sed/libexec/gnubin:/Library/Frameworks/Python.framework/Versions/3.6/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Applications/Postgres.app/Contents/Versions/latest/bin:/usr/local/go/bin
9 verbose lifecycle zappr@0.1.0-alpha.1~start: CWD: /Users/lschulz/zalando/dev/zappr
10 silly lifecycle zappr@0.1.0-alpha.1~start: Args: [ '-c', 'NODE_ENV=production node dist/server/server.min.js' ]
11 silly lifecycle zappr@0.1.0-alpha.1~start: Returned: code: 1  signal: null
12 info lifecycle zappr@0.1.0-alpha.1~start: Failed to exec start script
13 verbose stack Error: zappr@0.1.0-alpha.1 start: `NODE_ENV=production node dist/server/server.min.js`
13 verbose stack Exit status 1
....
```

**Comment by lotharschulz** [comment]
👍

**Comment by lotharschulz** [comment]
@frometor @fokusferit please check

**Comment by frometor** [comment]
👍

**Comment by lotharschulz** [comment]
👍

**Comment by frometor** [comment]
👍

**Comment by fokusferit** [comment]
👍

**Pull request merged**